infra/profile.sh

```diff
@@ -11,37 +11,48 @@ trap cleanup EXIT
 # TODO: take in file glob as command line argument
 PROFILES=tests/small/add.bril
 
+RUNMODES=("nothing" "rvsdg-optimize")
+
 # create temporary directory structure necessary for bench runs
 mkdir -p ./tmp/bench
-mkdir ./tmp/hyperfine
 
 # bench will benchmark a single bril file, outputting its contents to ./tmp/bench/<profile_name>.json
 bench() {
-    profile=$1
+    profile="../../../$1"
 
     # strip the file path down to just the file name
     # TODO: profile name is not unique, generate a unique output path (it will be aggregated anyway)
-    profile_name=$(basename -- $profile)
+    profile_file=$(basename -- $profile)
+    profile_name="${profile_file%.*}"
+
+    mkdir ./tmp/bench/"$profile_name"
+    pushd ./tmp/bench/"$profile_name"
```

Comment:
Instead of using pushd, can the whole bash script stay in the same directory but use this path for `$out`?